runners/spark/src/main/java/org/apache/beam/runners/spark/io/TestStorageLevelPTransform.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.runners.spark.io;
+
+import org.apache.beam.sdk.transforms.PTransform;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PDone;
+
+/**
+ * Get RDD storage level (for testing purpose only).
+ * It's a hack used in the {@link org.apache.beam.runners.spark.translation.TransformTranslator}.
```

Comment:
How about something like: "Some features, such as StorageLevel, can only be tested via PTransforms and custom evaluators, so while this is a test-only evaluator it has to be a part of the TransformTranslator but should not be used in applicative pipelines."